python/example/example_requirements.txt

```diff
@@ -1,4 +1,5 @@
 # pip requirements file
+numpy!=1.24.0 # See https://github.com/numpy/numpy/issues/22826
```

Comment:
should we add this restriction also to the spack package?

Reply:
The restriction is only required for the examples (a single one in fact), which isn't a target in the spack package. So I don't think it's necessary.

Reply:
still, numpy is a dependency in the spack package when building with python, so would be nice to have imo

Reply:
This dependency is because Arbor exposes some data as numpy arrays (spike array). This won't trigger the big. The user would need to write code themselves that runs into it.

Reply:
ok let's leave it then.